feat: support hide my directories

kt286 opened the pull request:
support hide my directories

Log: support hide my directories

kt286 committed:
feat: support hide my directories

support hide my directories

Log: support hide my directories

itsXuSt commented on src/dde-file-manager-lib/models/computermodel.cpp:
```diff
@@ -61,14 +62,17 @@ ComputerModel::ComputerModel(QObject *parent)
     m_nitems = 0;
     m_appEntryWatcher.reset(fileService->createFileWatcher(nullptr, DUrl(APPENTRY_ROOT), this));
 
+    bool disabledMyDirectories = DFMApplication::genericAttribute(DFMApplication::GA_HiddenMyDirectories).toBool();
```
How should I make them re-display without restarting dde-file-manager after I changed the config?
Your solution is not COMPLETE so I will NOT approve to merge.
Maybe you should watch the change of the attribute and reload the model when the value changed, that will be more acceptable.

kt286 committed:
fix: reload computer model when the hide my directories setting changed

reload computer model when the hide my directories setting changed

Log: reload computer model when the hide my directories setting changed

kt286 committed:
fix: reload computer model when the hide my directories setting changed

reload computer model when the hide my directories setting changed

Log: reload computer model when the hide my directories setting changed

kt286 committed:
Merge branch 'master' into feat/hide-my-directories

itsXuSt commented:
features may not be approve, you could make an issue first and out POs will take consider of it.
after that your PR will be consider to merge in.
如 Deepin 社区规范，考虑到商业原因，feature 可能不会被接纳，不过可以先提 issue，产品会考虑需求建议，如果产品觉得可行的话到时候才可能考虑你的需求 PR。
